### PR TITLE
added sync disabled icon to be shown during migration

### LINKED ISSUE #13129: STOR-04: Design an icon for 'sync is disabled'
https://github.com/ankidroid/Anki-Android/blob/0458ee60d6e533d475970bf2e2baefa2bc044a9f/AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt#L715-L717

If sync is disabled due to a storage migration, we want to inform the user that this is occurring.

It is planned to display the progress of the storage migration if the icon is pressed when in this state

### PR DESCRIPTION
**Pull request by criticalAY**
## Purpose / Description
Fixes #13129

## Fixes
Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

## How Has This Been Tested?
tested on oneplus nord ce
testsync

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by BrayanDSO** [comment]
This icon's arrows look different from the default sync icon.

I'd rather if we made our own new icon similar to this, with a line crossing the sync icon we use.

**Comment by criticalAY** [comment]
The sync error icon is also in the same fashion as above and I took this one from the vector asset from where we are using the current sync icon and the sync error icon.

**Comment by lukstbit** [comment]
I think the added icon is ok although it isn't like the current one, the change is most likely due to the strike line which would be in an awkward position with our current icon.

My question is if the sync locked icon isn't better as the migration (b)locks the ability to sync:

**Comment by criticalAY** [comment]
I like this locked sync icon as well so I am in favor of using it as it looks better in my opinion. @david-allison  @BrayanDSO

**Comment by david-allison** [comment]
"Locked" looks good to me

**Comment by criticalAY** [comment]
Done! updated it with locked sync icon

**Comment by david-allison** [comment]
Do you have a sample before/after screenshot?

**Comment by criticalAY** [comment]
Screenshot_4
Screenshot_3

File: AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt [hunk 1]
```diff
@@ -716,7 +715,9 @@ open class DeckPicker :
                 BadgeDrawableBuilder.removeBadge(menuItem)
             }
         }
-        menuItem.iconAlpha = if (syncIcon == SyncIconState.Disabled) Themes.ALPHA_ICON_DISABLED_LIGHT else Themes.ALPHA_ICON_ENABLED_LIGHT
+        if (syncIcon == SyncIconState.Disabled) {
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt:718]: This should be in the when clause above where there is already and entry for SyncIconState.Disabled.

**Review by lukstbit — CHANGES_REQUESTED** [review]

**Review by lukstbit — APPROVED** [review]

**Comment by oakkitten** [comment]
Personal opinion: I wonder if we can rethink the icon a bit. I think wireframe icons look best when the stroke thickness is consistent. Currently the u-shaped part of the lock is too thin for my taste. Also, the arrows have a different shape and position.

Said that, I think that it's easier to understand than the crossed-out sync icon, even if that one looks better.

**Comment by david-allison** [comment]
@criticalAY Are you likely to have time to produce/find a better icon, given other work that's in progress currently?

**Comment by criticalAY** [comment]
Sure, I will provide with few options and then we can discuss and pick one

**Comment by mikehardy** [comment]
Perfect is the enemy of good enough. We need scoped storage done and this has been stalled on nitpicks for more than a month

**Review by mikehardy — APPROVED** [review]

**Pull request merged**